docs/release_notes/DRAFT_Release_notes_0.0.8.adoc

```diff
@@ -11,6 +11,7 @@
 This is needed for Unicredit that has unspecified required parameters.
 Their design is based on the premise of TPPs traversing exact links in the preceding response.
 Old methods without query parameters are deprecated.
+- Unicredit adapter is updated to its latest version
```

Comment:
I would ask you to specify the version supported by the adapter. Because it could be not true in the release date.

Reply:
got it

Reply:
done